src/Symfony/Component/Runtime/RuntimeInterface.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Runtime;
+
+/**
+ * Enables decoupling apps from global state.
+ *
+ * @author Nicolas Grekas <p@tchwork.com>
+ */
+interface RuntimeInterface
+{
+    /**
+     * Returns a closure that can build and return any object of yours that represents your app.
+     *
+     * @return \Closure(): object
+     */
+    public function resolve(\Closure $app): \Closure;
+
+    /**
+     * Returns a closure that runs the app and returns its exit status.
```

Comment:
What about adding a note the output of the one is the input of the other?

Reply:
I added some words about it, does it sounds good to you?